apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,141 @@
+  const intl = useIntl();
+
+  const location = (
+    <p>
+      {user.currentCity || intl.formatMessage(commonMessages.notFound)},{" "}
+      {intl.formatMessage(
+        user.currentProvince
+          ? getProvinceOrTerritory(user.currentProvince)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const telephone = (
+    <p>
+      {user.telephone ? (
+        <Link
+          external
+          href={`tel:${user.telephone}`}
+          aria-label={user.telephone.replace(/.{1}/g, "$& ")}
+        >
+          {user.telephone}
+        </Link>
+      ) : (
+        ""
+      )}
+    </p>
+  );
+
+  const email = (
+    <Link external href={`mailto:${user.email}`}>
+      {user.email}
+    </Link>
+  );
+
+  const citizenship = (
+    <p>
+      {intl.formatMessage(
+        user.citizenship
+          ? getCitizenshipStatusesAdmin(user.citizenship)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const preferredLang = (
+    <p>
+      {intl.formatMessage({
+        defaultMessage: "General Communication:",
+        id: "gU7Nup",
+        description: "Label for preferred language in profile details box.",
+      })}{" "}
+      {intl.formatMessage(
+        user.preferredLang
+          ? getLanguage(user.preferredLang)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const preferredLangForInterview = (
+    <p>
+      {intl.formatMessage({
+        defaultMessage: "Spoken Interviews:",
+        id: "UYtDDb",
+        description:
+          "Label for preferred lang for interviews in profile details box.",
+      })}{" "}
+      {intl.formatMessage(
+        user.preferredLanguageForInterview
+          ? getLanguage(user.preferredLanguageForInterview)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const preferredLangForExam = (
+    <p>
+      {intl.formatMessage({
+        defaultMessage: "Written Exams:",
+        id: "M8Gy1Q",
+        description:
+          "Label for preferred lang for exams in profile details box.",
+      })}{" "}
+      {intl.formatMessage(
+        user.preferredLanguageForExam
+          ? getLanguage(user.preferredLanguageForExam)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const profileDetails = [
+    location,
+    telephone,
+    email,
+    citizenship,
+    preferredLang,
+    preferredLangForInterview,
+    preferredLangForExam,
+  ];
+  return (
+    <Well
+      data-h2-display="base(flex)"
+      data-h2-align-items="base(center)"
+      data-h2-justify-content="base(space-evenly)"
+      data-h2-gap="base(0, x.5)"
+      data-h2-color="base(black)"
+      data-h2-margin-bottom="base(x1)"
+      data-font-size="base(caption)"
+    >
+      {profileDetails.map((detail, index) => (
+        <>
+          {detail}
+          {index + 1 !== profileDetails.length && (
+            <span aria-hidden>&bull;</span>
+          )}
+        </>
+      ))}
+    </Well>
```

Comment:
I'm not finding anything inherently wrong with the code here but I'm not sure the design translated well to the web 😢
 
![Screenshot 2023-12-21 094729](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/2c28a110-8463-4c88-aef6-723b8789da8d)

Reply:
Ya same here, maybe shrinking the font-size would help a bit. However, once the screen size drops to tablet it doesn't look amazing. What do you think @Jerryescandon?

Reply:
What if we prioritize the line break of each bullet first, and leave line break within the text last?
e.g.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/a73ded18-877e-4a08-a616-ab2c70c356e0)

Reply:
Okay hopefully everything looks right now:

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/db1e3be0-4f81-4dfa-86b3-49a59dbe9063)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/8f64ead2-f8fd-40e0-a046-ffe409ab83e5)

Reply:
Ship it